libraries/AP_Param/tools/eedump_apparam.c

```diff
@@ -111,14 +120,14 @@ main(int argc, char *argv[])
     for (;; ) {
         uint8_t size;
         var = (struct Param_header *)&eeprom[index];
-        if (var->key == _sentinal_key ||
+        if (var->key_low == _sentinal_key ||
             var->type == _sentinal_type) {
             printf("end sentinel at %u\n", index);
             break;
         }
         size = type_size(var->type);
         printf("%04x: type %u (%s) key %u group_element %u size %d value ",
-               index, var->type, type_names[var->type], var->key, var->group_element, size);
+               index, var->type, type_names[var->type], var->key_low, var->group_element, size);
```

Comment:
i think this should be (key_high<<8)|key_low

Reply:
Fixed. Doh. Sorry about that one. Should have tested it after my conversion as well to catch it.